packages/gcf-utils/package.json

```diff
@@ -49,7 +52,7 @@
     "typescript": "^3.8.3"
   },
   "engines": {
-    "node": ">= 10.13.0"
+    "node": "^10.17.0"
```

Comment:
Oh that's interesting!  @bcoe I've never seen proper semver used here before.  Is that vaid?

Reply:
Context: It was the only way I could get `gts` to work and not complain about `promise` in the `fs` package not being available/backported.

Reply:
I believe it can be any semver:

https://docs.npmjs.com/files/package.json#engines

In their npm example they use `~`.